GCRCatalogs/user_config.py

```diff
@@ -0,0 +1,40 @@
+from .user_config_mgr import UserConfigManager
+from argparse import ArgumentParser, RawTextHelpFormatter
+
+if __name__ == "__main__":
+    usage="""Directly manipulate items in user config"""
+
+    parser = ArgumentParser(description=usage,
+                            formatter_class=RawTextHelpFormatter)
+    parser.add_argument("operation", help="Operation to be performed for a particular key",
+                        choices=("set", "get", "del"))
+    parser.add_argument("key", help="Key on which operation is to be performed")
+    parser.add_argument("value", help="Value to use when operation is 'set'",
+                        default=None, nargs="?")
+
+    args = parser.parse_args()
+
+    umgr = UserConfigManager()
+    
+    if args.operation == 'get':
+        old = umgr.get(args.key)
+        if old:
+            print(f"Value of {args.key}: {old}")
+        else:
+            print(f"{args.key} is not in the user config")
```

Comment:
The reason I used the less idiomatic form is because it avoids reading in the config file an extra time. Alternatively UserConfigManager could keep a cached copy of the file, but that seems risky since a session lasts an indeterminate time and some other process can meanwhile modify the file.
I don't have an opinion about supporting Python < 3.6.    I can change those lines if you think it's advisable, but I see in setup.py

> 'Programming Language :: Python :: 3.6',

so maybe it's not an issue.

Reply:
Re: using `in` --- I am ok with the current implementation as it serves the common use cases, but perhaps we should be accurate in the print out message and say:

```python
     f"{args.key} is not in the user config or its value is None"
```

Re: f-string --- I am ok with dropping support for Python < 3.6. I just want to make sure we are making a deliberate decision rather than making it an implied consequence.